db/seeds.rb

```diff
@@ -1,7 +1,21 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+apartments = [
```

Comment:
It will be helpful to seed a user as well so that if you at some point drop your database and need to seed again, you will be creating a user as well.  If not, you may run into an error because of the relation between the tables.